### PR TITLE
Millie render panel overflow

### LINKED ISSUE #63: [Bug]: Render panel overflows
### What happened?

If we use a component (such as a video) which plays at full-screen size, we need to be able to confine it to the render window without overlapping the edges.

### What browsers do you see the problem on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What operating system do you see the problem on?

- [ ] Windows
- [ ] Mac
- [ ] Linux

### How did we discover the bug?

- [ ] Reported by participant
- [ ] Observed on Sentry
- [X] Found in dev

### Priority

Blocking some use

### PR DESCRIPTION
**Pull request by gumillie222**
fix #63

### CODE REVIEW

File: src/app/editor/components/RenderPanel.tsx [hunk 1]
```diff
@@ -1,10 +1,15 @@
-import React, { useEffect, useState, useContext } from 'react'
+import React, { useEffect, useState, createContext, useContext } from 'react'
 import dynamic from 'next/dynamic.js'
+import styled, { ExecutionProps, FastOmit } from 'styled-components';
```
Comment (JamesPHoughton) [inline src/app/editor/components/RenderPanel.tsx:3]: I'm not sure we need to bring in additional styling libraries - why can't we do this just with tailwind? I'd rather not introduce future confusion by having competing style styles...

File: src/app/editor/components/RenderPanel.tsx [hunk 2]
```diff
@@ -14,7 +19,11 @@ const Stage = dynamic(
   {
     ssr: false,
   }
-)
+);
+
+const StyledStage = styled(Stage)`
+  .min-w-sm.mx-auto.aspect-video.relative
```
Comment (JamesPHoughton) [inline src/app/editor/components/RenderPanel.tsx:25]: these styles have tailwind equivalents, lets just use them...

**Review by JamesPHoughton — DISMISSED** [review]

**Comment by shapeseas** [comment]
@gumillie222 is this PR ready to go? You'll need to merge main into your branch!

**Review by shapeseas — APPROVED** [review]
Looks good, James' feedback was resolved.

**Pull request merged**